chevah/python-package
# [#4020] Patch the HP-UX build to load the right libraries.

Scope
=====

The latest 2.7.13 build is failing on HPUX

Why we got this
=============

* No complete tests for the new distribution when used with our full build and test system


Changes
=======

Add a patch for site.py on HPUX

The tests were extended to run a full test for compat.

As a drive-by change I have documented the basic principles of generating a patch.


How to try and test the changes
===============================

reviewers: @dumol 

this depends on #77 . I have include it here as otherwise the build is blocked

check that changes make sense.

for now it is using a specific branch from chevah/compat.... but I hope that branch will be merged soon into master and then I will update the code to use master

thanks
